test/integration/document/Thumbnails.e2e.test.js

```diff
@@ -0,0 +1,99 @@
+// <reference types="Cypress" />
+describe('Preview Document Thumbnails', () => {
+    const token = Cypress.env('ACCESS_TOKEN');
+    const fileId = Cypress.env('FILE_ID_DOC');
+    const THUMBNAIL_SELECTED_CLASS = 'bp-thumbnail-is-selected';
+
+    /* eslint-disable */
+    const getThumbnail = (pageNum) => cy.get(`.bp-thumbnail[data-bp-page-num=${pageNum}]`);
+    const showDocumentPreview = (enableThumbnailsSidebar) => {
```

Comment:
Passing in an object with a named `enableThumbnailsSidebar` property makes this a bit easier to read.